WPF/View/Pages/WorkSpace.xaml.cs

```diff
@@ -186,6 +186,19 @@ private void AddTaskControls()
                 Grid.SetColumn(MyControl, rowData.StartDateCol);
                 Grid.SetColumnSpan(MyControl, rowData.ColSpan);
                 Grid.SetZIndex(MyControl, 100);
+                
+                                //Tyler K.
+                //Don't want task tooltips on the project "button". 
+                if(i > 0)
+                {
+                    MyControl.ToolTip = viewModel.TooltipData[i - 1].OutputToolTip();
+                }
+                //If we're at index 0, it'll populate the Project "button". Add a project tooltip. 
+                if(i == 0)
```

Comment:
Just a Nit Pick, else if rather than if would be cleaner. The comment "Don't want task tooltips on the project "button".", is inaccurate. We do want them, instead maybe put "Tooltips for task controls"